cmd/trust-manager/app/options/options.go

```diff
@@ -187,6 +194,14 @@ func (o *Options) addAppFlags(fs *pflag.FlagSet) {
 		"readiness-probe-path", "/readyz",
 		"HTTP path to expose the readiness probe server.")
 
+	fs.DurationVar(o.LeaseDuration,
+		"lease-duration", time.Second*60,
+		"Lease duration for leader election")
+
+	fs.DurationVar(o.RenewDeadline,
+		"renew-deadline", time.Second*40,
```

Comment:
```suggestion
		"leader-election-renew-deadline", time.Second*40,
```

To give more context to user-facing flags.